### PR TITLE
Improve Login flow

### PR DESCRIPTION
**Pull request by clahey**
login
logout

### CODE REVIEW

File: frontend/src/app/account/login/login.component.html [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
-        </div>
+      <label for="password">Password</label>
+      <input type="password" formControlName="password" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.password.errors }" />
+      <div *ngIf="submitted && f.password.errors" class="invalid-feedback">
+        <div *ngIf="f.password.errors.required">Password is required</div>
+      </div>
     </div>
     <div class="form-group">
-        <a routerLink="../forgot" class="btn btn-link">I forgot my username and/or password.</a>
+      <a routerLink="../forgot" class="forgot">I forgot my username and/or password.</a>
```
Comment (Gailbear) [inline frontend/src/app/account/login/login.component.html:31]: i think this probably has to go away or like, do something other than what it does. also i'm not quite sure what lol. that's more of a todo than a comment on this.

File: frontend/src/app/account/login/login.component.html [hunk 2]
```diff
@@ -1,25 +1,47 @@
...
+      </button>
     </div>
-</form>
+  </form>
+  <div class="footer-container">
+    <div class="footer">
+      Don’t have a membership?

+      <br>
+      <a href="http://arisia.org/Registration" target="_blank">Click here for information on attending the remote con</a>.
```
Comment (Gailbear) [inline frontend/src/app/account/login/login.component.html:44]: https

**Review by Gailbear — APPROVED** [review]
I think we probably also want to can the gradient, or mirror it in the behind pages. but yay later task!

**Comment by clahey** [comment]
door-dragon

File: frontend/src/app/_components/door-dragon/door-dragon.component.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    </div>
+    <div *ngSwitchCase="'programming'">
+      To access Arisia programs and events, you need to be logged into
+      the site so that we know you have a badge and have signed the
+      Code of Conduct. If you still need to
+      register, <a href="https://reg.arisia.org/kiosk/web_reg/"
+      target="_blank">you can do so here</a>.
+    </div>
+    <div *ngSwitch="'discord'">
+      To access the Arisia Discord server, you need to verify your
```
Comment (Gailbear) [inline frontend/src/app/_components/door-dragon/door-dragon.component.html:25]: you first need to join the arisia discord.

File: frontend/src/app/_components/door-dragon/door-dragon.component.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="comments" *ngSwitchCase="'social'">
+      If you already have a badge and need help, the Info Desk is
+      available on Zoom here!
+    </div>
+    <div class="comments" *ngSwitchCase="'programming'">
+      If you already have a badge and need help, the Info Desk is
+      available on Zoom here!
+    </div>
+    <div class="comments" *ngSwitch="'discord'">
+      Don’t have a Discord account? Head over to Discord using this
```
Comment (Gailbear) [inline frontend/src/app/_components/door-dragon/door-dragon.component.html:47]: this is full of lies.

**Review by Gailbear — APPROVED** [review]
other than the copy's accuracy, looks legit.

**Pull request merged**